archive/app/ui/NotFound.scala

```diff
@@ -1,14 +1,9 @@
 package ui
 
-import conf.Configuration
-import play.api.libs.json.{JsValue, Json}
 import rendering.Renderable
+import rendering.core.JavascriptProps
+import play.api.libs.json.{JsObject, JsValue, Json}
 
 object NotFound extends Renderable {
-  override def props: Option[JsValue] = Some(
-    Json.obj(
-      "beaconUrl" -> Configuration.debug.beaconUrl
-    )
-  )
+  override def props: Option[JsValue] = Some(JsObject(List()))
```

Comment:
why passing an empty props rather than `None`?

Reply:
Haha, I didn't think of that 😅

Reply:
Oh, I remember now, this property needs to map to an empty JSON object at least, or [`JavascriptRendering.getCommonProps(...)`](https://github.com/guardian/frontend/pull/17995/files#diff-68c19301733dc555c0f2aa32c2a3b32dR22) always returns `None`. Perhaps I'm doing something incorrectly in the latter?

Reply:
I see. Let's fix it in `getCommonProps` then and make sure we can append `default` even when the component specific props is `None`. Le me know if you need any help